pvsite_datamodel/sqlmodels.py

```diff
@@ -264,3 +267,30 @@ class StatusSQL(Base, CreatedMixin):
     status_uuid = sa.Column(UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
     status = sa.Column(sa.String(255))
     message = sa.Column(sa.String(255))
+
+
+class InverterSQL(Base, CreatedMixin):
+    """Class representing the inverters table.
+
+    Each InverterSQL row represents an inverter identified in
+    a successful Enode link.
```

Comment:
Is the `Enode link` part still relevant or is this table the "generic" inverter table (not necessarily linked to Enode)?

Reply:
No longer relevant, given the generic aspect of `client_id`.